tests/server.spec.js

```diff
@@ -544,8 +544,12 @@ afterAll(() => {
   // with their contents.
   function writeE2eResult(key) {
     const target = `${__dirname}/e2e-${key}-output.json`;
-    fs.writeFileSync(target, JSON.stringify(e2eOutput[key], null, 2) + "\n");
-    console.info(`E2e ${key} tests output written to ${target}.`);
+    if (e2eOutput[key].length === 0) {
```

Comment:
I believe this shouldn't happen anymore, but it's still good to have I guess, to prevent any future accidents ;)

Reply:
The PT elec is unused, I'll remove it in another PR